src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+enum class Prize(val countOfMatch: Int, private val rank: Int, val prizeMoney: Int) {
+    MATCH_NONE(0, 0, 0),
+    MATCH_ONE(1, 0, 0),
+    MATCH_TWO(2, 0, 0),
+    MATCH_THREE(3, 5, 5000),
```

Comment:
`100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
    MATCH_THREE(3, 5, 5_000),
```